h2o-algos/src/test/java/hex/coxph/CoxPHTest.java

```diff
@@ -1,23 +1,21 @@
 package hex.coxph;
 
 import hex.StringPair;
+import org.junit.Assume;
 import org.junit.BeforeClass;
 import org.junit.Test;
-import water.Key;
-import water.MRTask;
-import water.Scope;
-import water.TestUtil;
-import water.fvec.Chunk;
-import water.fvec.Frame;
-import water.fvec.NewChunk;
-import water.fvec.Vec;
+import water.*;
+import water.fvec.*;
+
+import java.util.HashSet;
+import java.util.Set;
 
 import static org.junit.Assert.*;
 
 public class CoxPHTest extends TestUtil {
 
   @BeforeClass
-  public static void setup() { stall_till_cloudsize(1); }
+  public static void setup() { stall_till_cloudsize(2); }
```

Comment:
intentional?

Reply:
no! thanks - this is also causing the tests to fail